internal/builder/packages.go

```diff
@@ -189,6 +195,7 @@ func BuildPackage(options BuildOptions) (string, error) {
 		return destinationDir, nil
 	}
 
+	logger.Infof("Validating package %s", destinationDir)
```

Comment:
Nit. Can you add an equivalent message for the zip validation? Should this be at the debug level?

Reply:
Added and changed to Debug :+1: